:memo: Clarify generic CDN setup (#2867)
platformsh/platformsh-docs: docs/src/environments/search-engine-visibility.md

Platform.sh can't guarantee that indexers follow the instructions.
If you're concerned about access, set up [HTTP access control](./http-access-control.md).

> If you are using a CDN, add a [custom domain](../domains/steps/_index.md) to your production site and ensure everything is configured for [TLS ownership verification](../domains/troubleshoot.md#ownership-verification).
> 
> To change the environment's visibility to search engines, follow these steps:

Reviewer: :hammer_and_wrench: 
```suggestion
You can change visibility to search engines for production environments with a [custom domain](../domains/steps/_index.md).
If you're using a CDN, make sure everything is configured for [TLS ownership verification](../domains/troubleshoot.md#ownership-verification).

To change your production environment's visibility to search engines, follow these steps:
```

+++

- Select the project where you want to change visibility.

> - From the **Environment** menu, select the Production environment you want to change.

Reviewer: ```suggestion
- From the **Environment** menu, select your production environment.
```

- Click {{< icon settings >}} **Settings**.
- In the row with **Hide from search engines**, click **Edit {{< icon chevron >}}**.
- Select or clear the **Hide from search engines** checkbox.

<--->

> 

Reviewer: :clipboard: 
```suggestion
```

+++
title=Using the CLI
+++

Run the following command:

```bash
platform environment:info --environment {{< variable "PRODUCTION_ENVIRONMENT_NAME" >}} restrict_robots true
```

{{< /codetabs >}}

## How it's done

When the **Hide from search engines** is activated,

> Search engines are turned away from environments by including a `X-Robots-Tag` header:

Reviewer 2: 📋 
```suggestion
search engines are turned away from environments by including a `X-Robots-Tag` header:
```

```txt
X-Robots-Tag: noindex, nofollow
```

That tells search engine indexers to not index these sites and not traverse links from these sites.

> This keeps non-Production sites out of search engine indexes.
> For non-Production sites it can't be turned off.

Reviewer: :hammer_and_wrench: This isn't the place for this information. Plus it doesn't work absolutely.
```suggestion
This helps keep non-Production sites out of search engine indexes.
```

Author: Why doesn't it work absolutely? Where would be the place for the information that it can't be turned off for non-prod?
